src/sentry/models/commitfilechange.py

```diff
@@ -40,3 +41,23 @@ class Meta:
     @staticmethod
     def is_valid_type(value: str) -> bool:
         return value in COMMIT_FILE_CHANGE_TYPES
+
+
+def process_resource_change(instance, **kwargs):
+    from sentry.integrations.github.integration import GitHubIntegration
+    from sentry.integrations.gitlab.integration import GitlabIntegration
+    from sentry.tasks.codeowners import code_owners_auto_sync
+
+    filepaths = list(
```

Comment:
You don't need to convert to a `list` if you're just going to do an inclusion check.